cmd/nvidia-ctk/cdi/generate/generate.go

```diff
@@ -214,6 +215,12 @@ func (m command) build() *cli.Command {
 				Destination: &opts.disabledHooks,
 				Sources:     cli.EnvVars("NVIDIA_CTK_CDI_GENERATE_DISABLED_HOOKS"),
 			},
+			&cli.BoolFlag{
+				Name:        "enable-chmod-hook",
+				Usage:       "Enable the chmod hook for device folder permissions. This hook is disabled by default.",
+				Destination: &opts.enableChmodHook,
+				Sources:     cli.EnvVars("NVIDIA_CTK_CDI_GENERATE_ENABLE_CHMOD_HOOK"),
+			},
```

Comment:
Question: Why not have a `--force-hook[s]` option that overrides the `--disable-hook[s]` option above instead of implementing a single flag for each hook?